data/actionsmap/yunohost.yml

```diff
@@ -925,6 +925,54 @@ monitor:
 
 
 #############################
+#         Settings          #
+#############################
+settings:
+    category_help: Manage YunoHost global settings
+    actions:
+
+        ### settings_list()
+        list:
+            action_help: list all entries of the settings
+            api: GET /settings
+
+        ### settings_get()
+        get:
+            action_help: get an entry value in the settings
+            api: GET /settings/<key>
+            arguments:
+                key:
+                    help: Settings key
+
+        ### settings_set()
+        set:
```

Comment:
If this is meant to be used by end users somehow, shall we have a system of hooks or actions triggered somehow ? E.g. let's imagine there's a setting to define the SSL security policy (bigger/smaller key size), when does the effective change happens ?